src/Http/Controllers/Admin/FaqQAController.php

```diff
@@ -90,7 +90,9 @@ public function edit(FaqPresenter $presenter, $id)
      */
     public function status(FaqPresenter $presenter, $id)
```

Comment:
Let's name method `changeStatus()` and let's receive status as a second route param. Not in body.